js/app.js

```diff
@@ -4,6 +4,72 @@ $(document).ready( function() {
   // check to make sure JS is loaded
   console.log('JS is loaded!');
 
+
   // your code here
 
-});
+  // Commented out lines were plans that I couldn't get working,
```

Comment:
Anthony,

This is awesome. I love that you tried to add the radio buttons to select a type - no worries that you ran out of time finishing that feature, it's cool you even tried! 

The rest of code looks really good, great job! 

Matt